src/qumin/representations/paradigms.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this an object oriented implementation of formerly existing functionalities, or is there anything new here ?